webapp/channels/src/components/copy_url_context_menu/copy_url_context_menu.tsx

```diff
@@ -29,46 +29,52 @@ type Props = {
     };
 }
 
-export default class CopyUrlContextMenu extends React.PureComponent<Props> {
-    copy = () => {
-        let link = this.props.link;
+const CopyUrlContextMenu = ({
+    link,
+    siteURL,
+    actions,
+    menuId,
+    children,
+}: Props) => {
+    const copy = () => {
```

Comment:
Let's use here a `useCallback` to avoid passing a new function to the child on every render.